initializr-service/src/main/resources/application.yml

```diff
@@ -477,6 +477,16 @@ initializr:
           groupId: io.zipkin.java
           artifactId: zipkin-server
           starter: false
+    - name: Cloud Task
+      bom: cloud-bom
+      versionRange: 1.3.6.RELEASE
```

Comment:
Are you sure about the version range there? Do you specifically require Spring Boot `1.3.6` or an earlier version would work?